content/dstu2/document-reference.md

```diff
@@ -110,7 +110,43 @@ present.
    x-xss-protection → 1; mode=block
 </pre>
 
+## Operation docref
+
+Argonaut operation for querying DocumentReferences for the supplied parameters:
+
+    GET /DocumentReference/$docref?:parameters
+
+### Terminology Bindings
+
+<%= terminology_table(:document_reference_docref, :dstu2) %>
+
+### Parameters
+
+ Name     | Required? | Type          | Description
+----------|-----------|---------------|-------------------------------------------------
+`patient` | Yes       | [`reference`] | A reference to the patient whose document references are required. Example: `14067892`
+`type`    | Yes       | [`token`]     | The document reference type, can be a list of comma separated values. Example: `http://loinc.org|34133-9`
+`start`   | No        | [`date`]      | The start of the date range form which document reference records are required. If not provided, then all records upto the end or current date are considered. Example: `2014-09-24T12:00:00.000Z`
+`end`     | No        | [`date`]      | The end of the date range till which document reference records are required. If not provided, then all records upto the current date are considered. Example: `2016-09-24T12:00:00.000Z`
+
+Notes:   
```

Comment:
Need to call out that we don't support the relatesTo modifier elements. See http://localhost:3000/dstu2/patient/#retrieve-by-id for example